tools/ClassGen/InstrGen.cpp

```diff
@@ -147,8 +147,7 @@ int main(int argc, char **argv) {
   BB.newInstr("SoftMax")
       .addOperand("Dest", OperandKind::Out)
       .addOperand("Src", OperandKind::In)
-      .inplaceOperand({"Dest", "Src"})
-      .autoVerify(VerifyKind::SameType, {"Dest", "Src"})
+      .autoVerify(VerifyKind::NoVerify)
```

Comment:
Can we do `VerifyKind::SameShape` here instead? I'd like to avoid `NoVerify` as much as possible.

Reply:
Done. I added the shape checking to `InstrGen.cpp`.